examples/lock-app/nrfconnect/README.md

```diff
@@ -101,135 +100,116 @@ the nRF52840's NFC antenna. Moreover in the reaction on sensing field from the
 smartphone CHIP device will start BLE advertisement, what is an alternative way
 to trigger this from manually pressing Button 4.
 
-<a name="building"></a>
+## Setting up environment
 
-## Building
+First, checkout CHIP repository and sync submodules using the following command:
```

Comment:
Hmm with docker, this step can be skipped actually. But don't have a strong opinion if it's worth to mention this.